scripts/migrations/00001_initial/index.sql

```diff
@@ -13,26 +13,50 @@ CREATE TABLE IF NOT EXISTS backup_configs (
 
 DROP TABLE IF EXISTS backups CASCADE;
 CREATE TABLE IF NOT EXISTS backups (
-  id INTEGER PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
-  backup_configs_id INTEGER NOT NULL,
+  id INTEGER PRIMARY KEY,
+  backup_config_id INTEGER NOT NULL,
+  repository_status TEXT DEFAULT 'not-started' CHECK (
+    repository_status IN (
+      'not-started',
+      'in-progress',
+      'failed',
+      'success'
+    )
+  ),
   repository_cid TEXT,
+  blobs_status TEXT DEFAULT 'not-started' CHECK (
+    blobs_status IN (
+      'not-started',
+      'in-progress',
+      'failed',
+      'success'
+    )
+  ),
   blobs_cid TEXT,
+  preferences_status TEXT DEFAULT 'not-started' CHECK (
+    preferences_status IN (
+      'not-started',
+      'in-progress',
+      'failed',
+      'success'
+    )
+  ),
   preferences_cid TEXT,
   created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL,
-  FOREIGN KEY (backup_configs_id) REFERENCES backup_configs(id)
+  FOREIGN KEY (backup_config_id) REFERENCES backup_configs(id)
 );
 
 DROP TABLE IF EXISTS auth_sessions;
 CREATE TABLE IF NOT EXISTS auth_sessions (
-  key TEXT PRIMARY KEY,
+  KEY TEXT PRIMARY KEY,
```

Comment:
don't think this should be uppercase right?

```suggestion
  key TEXT PRIMARY KEY,
```

Reply:
Huh. That’s weird, I didn’t touch those lines manually. Good catch!